### PR TITLE
Added Bintray badge to release notes

### PR DESCRIPTION
**Pull request by NagRock**

### CODE REVIEW

File: src/main/groovy/org/mockito/release/notes/format/DetailedFormatter.java [hunk 1]
```diff
@@ -53,13 +53,20 @@ public String formatReleaseNotes(Collection<ReleaseNotesData> data) {
...
+    static String releaseSummary(String version, Date date, ContributionSet contributions, Map<String, Contributor> contributors,
                                  String vcsCommitsLink, String publicationRepository) {
         return authorsSummary(contributions, contributors, vcsCommitsLink) +
-                " - *" + DateFormat.formatDate(date) + "*" + " - published to " + publicationRepository + "\n" +
+                " - *" + DateFormat.formatDate(date) + "*" + " - published to " + getBintrayBadge(version, publicationRepository) + "\n" +
                 authorsSummaryAppendix(contributions, contributors);
     }
 
+    private static String getBintrayBadge(String version, String publicationRepository) {
+        StringBuilder sb = new StringBuilder();
```
Comment (wwilk) [inline src/main/groovy/org/mockito/release/notes/format/DetailedFormatter.java]: No need to use StringBuilder here, regular string concatenation would be more readable. Also 
sb.append("[![Bintray]").append("(https://img.shields.io/badge/Bintray-") can be replaced with one literal.

Comment (mstachniuk) [inline src/main/groovy/org/mockito/release/notes/format/DetailedFormatter.java]: Or use String.format("[Bintray] ....", version, ...) - up to you what will be more readable.
String concatenation with `+` is compiled by `javac` to `StringBuilder` version

File: src/main/groovy/org/mockito/release/notes/format/DetailedFormatter.java [hunk 2]
```diff
@@ -53,13 +53,20 @@ public String formatReleaseNotes(Collection<ReleaseNotesData> data) {
         return sb.toString().trim();
     }
 
-    static String releaseSummary(Date date, ContributionSet contributions, Map<String, Contributor> contributors,
+    static String releaseSummary(String version, Date date, ContributionSet contributions, Map<String, Contributor> contributors,
                                  String vcsCommitsLink, String publicationRepository) {
         return authorsSummary(contributions, contributors, vcsCommitsLink) +
-                " - *" + DateFormat.formatDate(date) + "*" + " - published to " + publicationRepository + "\n" +
+                " - *" + DateFormat.formatDate(date) + "*" + " - published to " + getBintrayBadge(version, publicationRepository) + "\n" +
```
Comment (mstachniuk) [inline src/main/groovy/org/mockito/release/notes/format/DetailedFormatter.java]: And what with Maven Central Badge?

Comment (szczepiq) [inline src/main/groovy/org/mockito/release/notes/format/DetailedFormatter.java]: Release notes formatter is not aware where the release is synced to. I think we can merge this code and start thinking how to model Maven Central release better.

**Comment by NagRock** [comment]
Sample generated release note:

**0.15.5** - [2 commits](https://github.com/mockito/mockito-release-tools-example/compare/v0.15.4...v0.15.5) by mkuster - *2017-05-27* - published to [Bintray](https://bintray.com/shipkit/examples/basic/0.15.5)
 - No pull requests referenced in commit messages.

**Comment by szczepiq** [comment]
I'm assuming that what @wwilk request has been taken care of one way or another. Thank you for writing a comment with how the release notes look after the change. It makes it super easy to review the change!!!

**Pull request merged**